tests/extra_source_and_tests/test_2d_T_pipe_VIPO_shell/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ ADD_EXECUTABLE(${PROJECT_NAME} ${EXECUTABLE_OUTPUT_PATH} ${DIR_SRCS})
 add_test(NAME ${PROJECT_NAME} COMMAND ${PROJECT_NAME} --state_recording=${TEST_STATE_RECORDING}
     WORKING_DIRECTORY ${EXECUTABLE_OUTPUT_PATH})
 
-set_tests_properties(${PROJECT_NAME} PROPERTIES LABELS "fluid_shell_interaction_VIPO")
+set_tests_properties(${PROJECT_NAME} PROPERTIES LABELS "pressure boundary")
```

Comment:
Just curious what is the function of changing the CMakeList file?

Reply:
This a ctest parameter. With argument -L and the key word such as pressure,  ctest wil only test all the tests with this key word.